JuliaDiff/Project.toml

```
@@ -0,0 +1,3 @@
+[deps]
+Franklin = "713c75ef-9fc9-4b05-94a9-213340da978e"
+NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"
```

```suggestion
NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"

```

just new line at the end (ignore if file is generated)